docs/features/DefaultInterfaceImplementation.md

```diff
@@ -43,16 +43,41 @@ class Test1 : I1
 
 - Declaring types within interfaces. **Protected** and **protected internal** accessibility is not supported.
 
-- Implementing interface methods in derived interfaces by using explicit implementation syntax, accessibility is private, allowed modifiers: **extern** and **async**.
+- Implementing interface methods in derived interfaces by using explicit implementation syntax, accessibility matches accessibility of the implemented member, allowed modifiers: **extern** and **async**.
 
-- Implementing interface properties and indexers in derived interfaces by using explicit implementation syntax, accessibility is private, allowed modifiers: **extern**.
+- Implementing interface properties and indexers in derived interfaces by using explicit implementation syntax, accessibility matches accessibility of the implemented member, allowed modifiers: **extern**.
 
-- Implementing interface events in derived interfaces by using explicit implementation syntax, accessibility is private, no allowed modifiers.
+- Implementing interface events in derived interfaces by using explicit implementation syntax, accessibility matches accessibility of the implemented member, no allowed modifiers.
 
 - Declaring static fields, auto-properties and field-like events (**protected** modifier is not allowed).
```

Comment:
>**protected** modifier is not allowed [](start = 66, length = 37)

This constraint is not correct.  Just as `protected static` fields are permitted in classes, they should be permitted in interfaces.

Reply:
> This constraint is not correct.  Just as protected static fields are permitted in classes, if fields et al are permitted at all, then protected versions of them should be permitted in interfaces. However the current spec forbids these categories of symbols.

So far, we didn't define what protected access means for interface members. This document accurately reflects that. We can revisit this at LDM.



---
In reply to: [255758478](https://github.com/dotnet/roslyn/pull/33281#discussion_r255758478) [](ancestors = 255758478)